### PR TITLE
docs: Relayer

### PR DESCRIPTION
**Pull request by alexgao001**
## Description

docs for mainnet relayer

## Rationale

tell us why we need these changes...

### CODE REVIEW

File: docs/guide/greenfield-blockchain/run-node/run-mainnet-challenger.md [hunk 1]
```diff
@@ -1,163 +1,9 @@
 ---
-title: Run Testnet Relayer and Challenger
-order: 5
+title: Run Mainnet Challenger
```
Comment (KeefeL) [inline docs/guide/greenfield-blockchain/run-node/run-mainnet-challenger.md:2]: This doc should include both mainnet and testnet challenger, so the title should be Run Challenger?

File: docs/guide/greenfield-blockchain/run-node/run-mainnet-relayer.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
+---
+title: Run Mainnet Relayer
```
Comment (KeefeL) [inline docs/guide/greenfield-blockchain/run-node/run-mainnet-relayer.md:2]: This doc should include how to run relayer on Mainnet and Testnet, so the title and the file name better change to Run Relayer?

File: docs/guide/greenfield-blockchain/run-node/run-mainnet-challenger.md (hunk 1 above)
Comment (alexgao001) [reply docs/guide/greenfield-blockchain/run-node/run-mainnet-challenger.md]: fixed

File: docs/guide/greenfield-blockchain/run-node/run-mainnet-relayer.md (hunk 2 above)
Comment (alexgao001) [reply docs/guide/greenfield-blockchain/run-node/run-mainnet-relayer.md]: fixed

**Review by KeefeL — APPROVED** [review]

**Pull request merged**